docs/en/install-upgrade/installing-stack-demo-self.asciidoc

```diff
@@ -0,0 +1,713 @@
+----
+network.host: 10.128.0.132
+----
+
+. Save your changes and close the editor.
+
+. Start {es} on the second node:
++
+[source,"shell"]
+----
+sudo systemctl start elasticsearch.service
+----
+
+. **Optionally**, to view the progress as the second {es} node starts up and connects to the first {es} node, open a new terminal into the second node and `tail` the {es} log file:
++
+[source,"shell"]
+----
+sudo tail -f /var/log/elasticsearch/elasticsearch-demo.log
+----
++
+Notice in the log file some helpful diagnostics, such as:
++
+* `Security is enabled`
+* `Profiling is enabled`
+* `using discovery type [multi-node]`
+* `intialized`
+* `starting...`
++
+After a minute or so, the log should show a message like:
++
+[source,"shell"]
+----
+[<hostname2>] master node changed {previous [], current [<hostname1>...]}
+----
++
+Here, `hostname1` is your first {es} instance node, and `hostname2` is your second {es} instance node.
++
+The message indicates that the second {es} node has successfully contacted the initial {es} node and joined the cluster.
+
+. As a final check, run the following `curl` request on the new node to confirm that {es} is still running properly and viewable at the new node's `localhost` IP address. Note that you need to replace `$ELASTIC_PASSWORD` with the same `elastic` superuser password that you used on the first {es} node.
++
+["source","sh",subs="attributes"]
+----
+sudo curl --cacert /etc/elasticsearch/certs/http_ca.crt -u elastic:$ELASTIC_PASSWORD https://localhost:9200
+----
++
+["source","js",subs="attributes,callouts"]
+----
+{
+  "name" : "Cp9oae6",
+  "cluster_name" : "elasticsearch",
+  "cluster_uuid" : "AT69_C_DTp-1qgIJlatQqA",
+  "version" : {
+    "number" : "{version_qualified}",
+    "build_type" : "{build_type}",
+    "build_hash" : "f27399d",
+    "build_flavor" : "default",
+    "build_date" : "2016-03-30T09:51:41.449Z",
+    "build_snapshot" : false,
+    "lucene_version" : "{lucene_version}",
+    "minimum_wire_compatibility_version" : "1.2.3",
+    "minimum_index_compatibility_version" : "1.2.3"
+  },
+  "tagline" : "You Know, for Search"
+}
+----
+
+[discrete]
+[[install-stack-self-elasticsearch-third]]
+== Step 5: Set up a third {es} node
+
+To set up your third {es} node, follow exactly the same steps as you did previously in <<install-stack-self-elasticsearch-second>>. The process is identical for each additional {es} node that you would like to add to the cluster. As a recommended best practice, create a new enrollment token for each new node that you add.
+
+[discrete]
+[[install-stack-self-kibana]]
+== Step 6: Install {kib}
+
+As with {es}, you can use RPM to install {kib} on another host. You can find details about all of the following steps in the section {kibana-ref}/rpm.html#install-rpm[Install {kib} with RPM].
+
+. Log in to the host where you'd like to install {kib} and create a working directory for the installation package:
++
+["source","shell"]
+----
+mkdir elastic-install-files
```

Comment:
Should probably be `kibana-install-files` for consistency with the service we're installing.